tests/Infrastructure/Vine/GetActiveFiltersTest.php

```
@@ -0,0 +1,197 @@
+                [],
+                []
+            );
+
+            $this->assertEquals(0, $activeFilters->total());
+        }
+    }
+
+    public function test_it_returns_only_scoped_if_active_taxa_do_not_exist()
+    {
+        foreach (Catalog::drivers() as $catalog) {
+            $taxonomy = $catalog->createTaxonomy();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$taxonomy'. (undefined)

(UnusedLocalVariable)